spec/system/candidate_interface/candidate_account_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+
+RSpec.feature 'Candidate account' do
+  scenario 'Candidate signs up, out, and in again' do
+    given_i_am_a_candidate_without_an_account
+
+    when_i_visit_the_signup_page
+    and_i_submit_my_email_address
+    then_i_receive_an_email_with_a_signin_link
```

Comment:
Should this not be a sign up link?
Are sign up and sign in email content going to be the same?

Reply:
Updated!